### ISSUE TITLE
orcid: search "also known as"?

### ISSUE THREAD

**Issue by yarikoptic** [con/tributors#46]
In regards to https://github.com/datalad/datalad/pull/4789/files#diff-c382e6d696dba7770df288b9f60d3b45R68 where I spotted that a likely orcid was not added: I think this one https://orcid.org/0000-0001-7037-2449 . He does have the name ("Horea Christian") in "Also known as". Could that be searched for as well?

**Comment by vsoch** [comment]
Just tested this - I'm not sure it helps because it gives us over 400 results, even if the first is correct it's not guaranteed to be.

```
WARNING:github:Found 483 results for Christian, Horea:

0000-0001-7037-2449
0000-0001-6339-4623
0000-0001-7150-8029
0000-0002-5774-8234
0000-0001-5116-2147
0000-0002-0914-5172
0000-0001-8144-6570
0000-0002-3329-2949
0000-0002-5295-3757
0000-0002-4815-2957
0000-0002-5046-5477
0000-0002-6937-0486
0000-0003-4777-7821
0000-0003-2051-3973
0000-0002-0255-9364
0000-0002-4870-8249
0000-0003-1711-3370
0000-0003-2415-5130
0000-0002-3432-9168
0000-0002-2921-1843
0000-0003-3671-3048
0000-0003-0907-9774
0000-0002-0930-441X
0000-0002-9666-2839
0000-0002-8627-5704
0000-0002-9514-6234
0000-0002-7260-6226
0000-0002-3987-9089
0000-0003-3318-8014
0000-0003-3821-0356
0000-0002-5237-1562
0000-0002-5133-5746
0000-0001-5076-8518
0000-0003-2260-5046
0000-0002-9538-8583
0000-0002-3188-094X
0000-0001-8577-4467
0000-0001-6755-3492
0000-0002-4450-4572
0000-0003-3473-6393
0000-0002-0682-8808
0000-0002-6176-1669
0000-0001-5701-4006
0000-0001-9127-8136
0000-0001-8099-0507
0000-0002-4259-9299
0000-0001-9199-8627
0000-0002-9533-688X
0000-0002-5445-8290
0000-0002-3339-1306
0000-0003-3110-3857
0000-0003-4818-207X
0000-0002-8468-0829
0000-0002-2939-9989
0000-0002-5691-5248
0000-0001-8639-9398
0000-0003-4961-9421
0000-0002-5644-7229
0000-0001-5324-8331
0000-0001-9218-5564
0000-0002-3178-4042
0000-0001-7342-1728
0000-0002-6574-0988
0000-0001-7794-8375
0000-0001-5475-3938
0000-0001-7753-7090
0000-0002-6326-7648
0000-0001-8761-693X
0000-0002-2703-6869
0000-0002-6944-5601
0000-0003-0967-4530
0000-0002-5620-7727
0000-0002-6954-6859
0000-0001-7920-4257
0000-0002-2918-5550
0000-0001-6502-7114
0000-0001-5550-3340
0000-0003-4009-6468
0000-0002-1718-1827
0000-0001-5105-8489
0000-0001-8402-6854
0000-0001-5294-9451
0000-0002-4792-7036
0000-0003-3228-7862
0000-0003-2771-8752
0000-0002-7229-6942
0000-0003-3211-3061
0000-0001-5674-8112
0000-0003-4937-2625
0000-0001-7685-3497
0000-0001-9404-2241
0000-0001-9358-4989
0000-0003-4231-625X
0000-0002-7962-0231
0000-0001-7216-6200
0000-0001-8848-5775
0000-0003-2513-2563
0000-0002-9832-2934
0000-0001-8482-0345
0000-0002-6226-8251
0000-0002-7933-2424
0000-0003-4722-3044
0000-0002-8715-2714
0000-0003-2874-1161
0000-0002-4728-6398
0000-0003-3714-5089
0000-0002-8029-0866
0000-0001-9690-1417
0000-0002-7441-5941
0000-0002-3165-4002
0000-0002-3336-5853
0000-0001-5473-7508
0000-0001-7968-0851
0000-0002-5204-2443
0000-0002-4098-8178
0000-0003-3231-0962
0000-0003-4993-2055
0000-0002-1112-4936
0000-0002-1399-8020
0000-0003-3744-0977
0000-0002-1395-2962
0000-0003-0502-3693
0000-0001-6303-621X
0000-0002-5221-0531
0000-0002-1508-6404
0000-0003-4312-7629
0000-0002-7831-331X
0000-0003-4934-1028
0000-0002-7424-3983
0000-0002-1178-5232
0000-0002-6731-5168
0000-0001-9570-5692
0000-0001-6080-7533
0000-0001-8758-3306
0000-0001-9718-1394
0000-0002-6936-6444
0000-0003-3224-6841
0000-0002-3954-6583
0000-0003-4940-0868
0000-0002-1596-5599
0000-0002-9887-0092
0000-0001-5555-4965
0000-0003-3109-4305
0000-0003-3037-7506
0000-0003-0061-7687
0000-0003-1299-7365
0000-0001-9267-5379
0000-0001-5668-2344
0000-0003-1382-1978
0000-0002-6660-5251
0000-0001-7399-6003
0000-0003-4361-9041
0000-0002-1951-2503
0000-0002-9294-2323
0000-0002-6911-698X
0000-0002-7440-3270
0000-0002-9825-170X
0000-0002-7349-4000
0000-0002-1235-9585
0000-0002-3826-0989
0000-0001-5704-2234
0000-0003-2839-8949
0000-0001-5404-5536
0000-0002-5615-2113
0000-0003-3477-2992
0000-0003-4875-391X
0000-0003-1102-4158
0000-0003-4029-4691
0000-0001-9226-780X
0000-0001-5094-645X
0000-0002-4748-2129
0000-0003-2134-3937
0000-0002-2924-7547
0000-0002-2386-1626
0000-0002-8619-0444
0000-0002-5839-1468
0000-0003-3522-9182
0000-0002-4818-6006
0000-0002-2538-7374
0000-0002-5460-7698
0000-0002-6026-5858
0000-0001-7524-7209
0000-0001-7309-1167
0000-0001-6834-7121
0000-0001-9119-8128
0000-0003-2969-2445
0000-0002-6707-1351
0000-0002-1098-4256
0000-0003-0091-9650
0000-0002-2787-052X
0000-0003-0406-3319
0000-0003-3371-6593
0000-0002-9009-1916
0000-0002-5724-495X
0000-0002-1178-2429
0000-0003-3703-5840
0000-0003-0957-9664
0000-0002-9307-8629
0000-0002-7350-9049
0000-0002-3822-7097
0000-0001-8977-2063
0000-0001-6501-0072
0000-0001-7277-4505
0000-0001-6019-8838
0000-0003-4338-3526
0000-0001-7967-9156
0000-0002-9057-5507
0000-0002-4369-6285
0000-0002-1560-673X
0000-0003-3040-0382
0000-0002-8183-532X
0000-0001-8325-2737
0000-0001-7994-2962
0000-0002-2408-4064
0000-0003-3862-8525
0000-0002-0657-9143
0000-0003-3079-2494
0000-0001-5140-226X
0000-0002-0217-734X
0000-0003-1927-8064
0000-0001-7266-5544
0000-0003-0603-831X
0000-0002-7892-4815
0000-0002-2294-6265
0000-0002-3708-5859
0000-0001-5855-9198
0000-0002-3795-6749
0000-0001-7872-9617
0000-0002-9194-1740
0000-0002-7378-6919
0000-0003-1475-4113
0000-0002-1002-6912
0000-0003-2460-1997
0000-0001-8433-1226
0000-0002-8883-0038
0000-0002-3178-0906
0000-0002-4458-0700
0000-0003-2652-5035
0000-0001-9672-0093
0000-0002-8642-4714
0000-0002-4209-8152
0000-0003-4095-0572
0000-0003-1895-987X
0000-0002-5710-2310
0000-0002-5186-4445
0000-0003-3566-1020
0000-0002-6094-5167
0000-0003-3302-0780
0000-0003-1966-2680
0000-0001-5605-8437
0000-0002-4818-4350
0000-0002-1779-0972
0000-0003-2081-1743
0000-0003-4153-3252
0000-0002-8785-6252
0000-0001-5764-8164
0000-0002-2560-2556
0000-0002-7754-8395
0000-0002-7551-2460
0000-0003-3531-6606
0000-0002-4072-6573
0000-0002-1685-1657
0000-0002-1013-6484
0000-0003-2976-3653
0000-0001-8618-0337
0000-0001-8401-3981
0000-0001-9665-8162
0000-0003-4507-1657
0000-0001-7095-8829
0000-0001-8741-4924
0000-0003-1943-838X
0000-0003-2176-4918
0000-0002-7742-486X
0000-0002-2438-610X
0000-0003-1148-4314
0000-0003-1236-0647
0000-0002-0075-0253
0000-0002-0248-8914
0000-0001-5702-0852
0000-0002-1541-6952
0000-0002-6575-356X
0000-0001-9164-188X
0000-0002-2626-2290
0000-0003-4865-3062
0000-0003-2980-729X
0000-0001-9455-4212
0000-0003-4160-7629
0000-0003-4629-8343
0000-0003-2850-5361
0000-0002-7462-6386
0000-0003-2064-9942
0000-0002-3827-0351
0000-0002-8872-9896
0000-0001-9619-7167
0000-0003-4452-5741
0000-0003-1532-203X
0000-0001-8550-0167
0000-0002-0330-0004
0000-0003-3012-2626
0000-0001-5472-7609
0000-0003-2754-9530
0000-0001-9801-2197
0000-0002-6633-7998
0000-0003-3667-7462
0000-0002-1743-816X
0000-0002-6170-1869
0000-0003-4120-5873
0000-0001-5329-475X
0000-0003-4276-1468
0000-0001-7712-478X
0000-0002-7506-5971
0000-0001-9009-719X
0000-0003-1080-9973
0000-0002-1117-1089
0000-0002-0573-4340
0000-0002-1497-0353
0000-0003-1874-9469
0000-0001-7156-5111
0000-0002-5511-8722
0000-0003-1361-5373
0000-0003-1141-261X
0000-0002-5817-6709
0000-0001-7033-4178
0000-0002-6913-4192
0000-0002-7351-8084
0000-0002-5519-8283
0000-0002-2358-004X
0000-0002-4104-3758
0000-0003-3805-8219
0000-0003-4768-5062
0000-0002-5468-401X
0000-0003-1496-7848
0000-0002-8768-2432
0000-0001-9659-9449
0000-0001-9589-1966
0000-0002-1251-9761
0000-0001-5366-5039
0000-0003-0918-9375
0000-0002-3266-5061
0000-0002-6585-3739
0000-0002-3103-1243
0000-0003-2052-359X
0000-0001-8726-4911
0000-0002-1899-0597
0000-0001-8930-6599
0000-0002-2327-1052
0000-0001-7253-3509
0000-0002-5495-6601
0000-0002-5832-349X
0000-0002-5237-9628
0000-0002-5363-6403
0000-0002-2937-7819
0000-0002-8894-1782
0000-0003-2993-4627
0000-0003-0333-9336
0000-0001-6536-4591
0000-0002-9595-0038
0000-0002-6750-6773
0000-0003-0971-0255
0000-0003-0413-3727
0000-0002-3658-9692
0000-0003-3024-3217
0000-0003-0970-7750
0000-0003-2792-6412
0000-0003-4735-6920
0000-0001-6249-9271
0000-0001-6281-6601
0000-0003-0224-4968
0000-0002-9067-2403
0000-0002-8701-4370
0000-0003-0025-2702
0000-0003-3825-2513
0000-0002-7666-3068
0000-0002-9988-6176
0000-0002-5316-993X
0000-0002-6245-6850
0000-0003-3845-6831
0000-0002-9559-0976
0000-0002-2808-8265
0000-0003-4851-7116
0000-0002-9903-3595
0000-0001-7335-4561
0000-0002-3645-4747
0000-0001-9367-6066
0000-0003-1789-0073
0000-0001-5675-0730
0000-0002-3745-8686
0000-0003-2399-2332
0000-0001-7241-3822
0000-0002-4184-0095
0000-0003-2801-4618
0000-0002-5338-0444
0000-0002-3029-173X
0000-0002-7027-1880
0000-0001-5500-6774
0000-0003-3054-2831
0000-0002-9799-7809
0000-0002-9884-1240
0000-0003-2982-9931
0000-0001-5709-9388
0000-0001-9286-8947
0000-0002-0437-1453
0000-0001-6700-5955
0000-0002-0468-9509
0000-0002-7091-5594
0000-0001-9251-476X
0000-0002-1551-9070
0000-0002-6138-6045
0000-0002-7040-2203
0000-0002-9061-8001
0000-0001-7645-7048
0000-0002-6869-3849
0000-0002-7015-7316
0000-0002-3865-0610
0000-0001-5788-8803
0000-0003-4496-1308
0000-0002-5390-9301
0000-0002-5916-4793
0000-0002-2140-1538
0000-0001-9517-2830
0000-0003-2234-7075
0000-0002-8215-7006
0000-0003-4222-8858
0000-0001-8649-2031
0000-0001-5118-0003
0000-0001-8918-5977
0000-0003-0525-3843
0000-0003-3642-3083
0000-0003-4529-4578
0000-0003-2441-0816
0000-0001-6114-0355
0000-0002-2005-7093
0000-0002-6821-6348
0000-0002-9759-1200
0000-0002-8177-4483
0000-0002-5249-8010
0000-0003-2700-7678
0000-0001-6950-546X
0000-0002-6510-7963
0000-0003-3249-0297
0000-0001-5421-4336
0000-0002-4785-8982
0000-0002-7770-9048
0000-0002-5276-8666
0000-0002-9341-854X
0000-0002-1272-3877
0000-0001-8074-770X
0000-0002-6087-0644
0000-0002-2340-808X
0000-0003-3177-0086
0000-0002-9786-8054
0000-0002-4741-8290
0000-0002-3382-7667
0000-0003-2560-6407
0000-0002-1948-6341
0000-0001-9624-2431
0000-0001-9494-5977
0000-0003-2263-8307
0000-0002-3833-9152
0000-0002-4841-7011
0000-0001-9779-6583
0000-0001-8919-4005
0000-0002-5866-3443
0000-0001-7485-0815
0000-0003-4928-6751
0000-0001-6242-187X
0000-0002-5080-036X
0000-0002-9545-5862
0000-0003-0511-0017
0000-0002-6718-1913
0000-0003-3155-2935
0000-0002-7702-8496
0000-0001-9381-633X
0000-0001-6733-0993
0000-0001-7690-1794
0000-0002-9676-6777
0000-0002-7336-2630
0000-0002-6100-9389
0000-0002-6970-7843
0000-0002-4947-3883
0000-0001-9596-8281
0000-0001-6261-7987
0000-0002-7503-8703
You should look these up and add the correct in your .tributors lookup
```

**Comment by vsoch** [comment]
Do you still want me to add it?

**Comment by vsoch** [comment]
See https://github.com/con/tributors/pull/48, I added it.

**Comment by vsoch** [comment]
This will be closed with #48 because we don't use specific attributes for the search anymore.
